README.md

```diff
@@ -0,0 +1,268 @@
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [JavaScript-Capstone] <a name="about-project"></a>
+
+<!-- > Describe your project in 1 or 2 sentences. -->
+
+**[JavaScript-Capstone]** JavaScript-Capstone is a collaborative project where we built our web application based on an external API. We select an API that provides data about movies and then build the WebApp using JavaScript.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<!-- > Describe the tech stack and include only the relevant sections that apply to your project. -->
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>Linters</li>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>Javascript</li>
+    <li>Webpack</li>
+    <li>Jest</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Github pages</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>No database</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+<!-- > Describe between 1-3 key features of the application. -->
+
+- **[Use a minimalist style to build the project]**
+- **[Best practices of ES6 syntax]**
+- **[Use of javascript to the functionalities of the page]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+<!-- > Add a link to your deployed project. -->
+
+- [JavaScript-Capstone(video)](https://drive.google.com/file/d/16A2lyinKh8LzZwWwKmuhI4ByM5XZtMLt/view?usp=sharing)
+
+<!-- <p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+<!-- > Describe how a new developer could make use of your project. -->
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need: 
+
+None
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone https://github.com/victormc13/JavaScript-Capstone.git
+```
+
+
+### Install
+
+Install this project with:
+
+No install require
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+- go to the folder where you cloned the repository
+- Open the index.html file with your favorite web browser
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+<!-- ### Run tests
+
+To run tests, run the following command: -->
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+<!-- ### Deployment
+
+You can deploy this project using: -->
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
```

Comment:
- [ **OPTIONAL** ] For clean code and best practices, I think it would be better if you remove all the commented and unused code lines from your codebase